vulnerable_people_form/form_pages/address_lookup.py

```diff
@@ -42,7 +43,7 @@ def get_address_lookup():
         }
         return redirect("/support-address")
 
-    prev_path = append_querystring_params("/nhs-letter" if is_nhs_login_user() else "/date-of-birth")
+    prev_path = append_querystring_params("/postcode-eligibility")
```

Comment:
has this been tested using an nhs login account to make sure the back behaviour works as expected?

Reply:
Ah, no I'll try and get that set up locally.